shuttleofx_render/cache.py

```diff
@@ -0,0 +1,69 @@
+import os
+from processify import processify
+
+import config
+
+@processify
+def cleanCache(cachePath):
+    '''
+        Clean the folder so it does not get bigger
+        than the limit set in the config file
+    '''
+    # Clean the render folder
+    allFiles = []
+    cacheSize = 0
+
+    # Loop through each files and delete the oldest
+    for path, dirs, files in os.walk(cachePath):
+        for file in files:
+            filePath = os.path.join(path, file)
+            fileSize = os.stat(filePath).st_size
+            # List all file : timestamp | size | filename
+            allFiles.append((os.path.getmtime(filePath), fileSize, filePath))
+            cacheSize += fileSize
+
+    allFiles.sort()
+
+    # TODO : Maybe pass the max size as an argument
+    while cacheSize >= config.cacheMaxSize:
+        for file in allFiles:
```

Comment:
You only need an `if: break` inside the `for` loop.
Here, as soon as the cache limit is exceeded you remove everything as you enter into the for loop which iterate on all files.